docs/docs/running-a-gatsby-preview-server.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Running a Gatsby Preview Server
+---
+
+When editing content in a CMS, it's great to see a preview of changes before they go live.
+
+This can be accomplished by setting up a preview server with Gatsby, for which there are two options:
+
+## Rolling your own Preview
+
+Roughly, you can follow these steps to enable your own preview server:
+
+- Run `gatsby develop` on a Node.js host like [Heroku](/docs/deploying-to-heroku/) with the [ENABLE_GATSBY_REFRESH_ENDPOINT environment variable](/docs/environment-variables/#reserved-environment-variables) enabled, which will configure the develop server to re-source data when it receives a `POST` request to `/__refresh`.
+- Then setup your CMS to ping this URL on the preview server whenever content is changed.
+
+## Using Gatsby Cloud Preview
+
+For a managed Preview server without having to create infrastructure yourself, try out [Gatsby Cloud](https://www.gatsbyjs.com/cloud). It's free for personal projects and single-purpose sites, with paid packages for a range of project and company needs.
```

Comment:
Worth mentioning anything about the CMSes it supports out of the box?

Reply:
That seems like potential for docs debt IMO, as every CMS they add would require an update to this page.

Reply:
Yup, good call.